ansible/roles/stack-badger/defaults/main.yml

```diff
@@ -1 +1,6 @@
 badger_admin_user: admin
+badger_replicas: 1
+badger_reservation_memory: 300MB
+badger_limit_memory: 500MB
+badger_reserve_cpu: 0.1
+badger_limit_cpu: 0.5
```

Comment:
remove the below binary file. I'm not able to comment on that file.. I meant rebase and delete